src/backend/bs_check_consistency

```diff
@@ -276,7 +277,7 @@ sub check_package_dir {
       # just a flag file
     } elsif ($ent eq "rpmlint.log") {
       # unchecked for now
-    } elsif ($ent eq ".preinstallimages") {
+    } elsif ($ent =~ /preinstall/) {
```

Comment:
this looks suspicious, it would also hit when someone builds an rpm call preinstall, right?

(this script is also used on production system to detect errors, not only in the test suite)